### ISSUE TITLE
Wrong fonts used when converting SVG to PNG.

### ISSUE THREAD

**Issue by yoopyman** [lovell/sharp#3693]
### Is this a possible bug in a feature of sharp, unrelated to installation?

- [X] Running `npm install sharp` completes without error.
- [X] Running `node -e "require('sharp')"` completes without error.

If you cannot confirm both of these, please open an [installation issue](https://github.com/lovell/sharp/issues/new?labels=installation&template=installation.md) instead.

### Are you using the latest version of sharp?

- [X] I am using the latest version of `sharp` as reported by `npm view sharp dist-tags.latest`.

If you cannot confirm this, please upgrade to the latest version and try again before opening an issue.

If you are using another package which depends on a version of `sharp` that is not the latest, please open an issue against that package instead.

### What is the output of running `npx envinfo --binaries --system --npmPackages=sharp --npmGlobalPackages=sharp`?

You are currently us
  System:
    OS: Windows 10 10.0.17763
    CPU: (4) x64 Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz
    Memory: 4.95 GB / 15.88 GB
  Binaries:
    Node: 20.2.0 - C:\tools\node\node.EXE
    npm: 9.6.6 - C:\tools\node\npm.CMD

### What are the steps to reproduce?

1) Load a SVG file.
2) Save it as PNG file.

### What is the expected behaviour?

The font from the PNG image to be the same as in the SVG file.

### Please provide a minimal, standalone code sample, without other dependencies, that demonstrates this problem

![input](https://github.com/lovell/sharp/assets/121875047/e78a4f27-73ce-423e-adae-d2730b142ad6)

var sharpConverter = Sharp(Buffer.from(svgContent));  
var pngContent       = await sharpConverter.toFormat('png').toBuffer();

![output](https://github.com/lovell/sharp/assets/121875047/fd0797c8-6f83-46f8-85b9-7d58ca5a95a8)

### Please provide sample image(s) that help explain this problem

The problem what I observed is the FontConfig part from libvips-42.dll that doesn't use the correct fonts.
I had made a debug on the font configuration and attached as a file.

[font_config_debug.log](https://github.com/lovell/sharp/files/11689048/font_config_debug.log)

He tries to use the "zh-tw" family lang and matches the font to "MingLiU-ExtB". 
It looks like it's something hardcoded in the libvips binaries.

I have tested also on another computer this time it was Windows Server 2022 and the issue was still present.
Also tried to set the environment variables for lang and still it didn't work: 
   LC_ALL=en
   LANG=en
   FC_LANG=en
   LC_CTYPE=en

After applying some restriction in the fonts.conf **it works correctly**: 

```html
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
<fontconfig>
 <dir>c:/Windows/Fonts/</dir>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>MingLiU-ExtB</string></patelt>
  </rejectfont>
 </selectfont>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>MingLiU</string></patelt>
  </rejectfont>
 </selectfont>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>PMingLiU</string></patelt>
  </rejectfont>
 </selectfont>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>MingLiU_HKSCS</string></patelt>
  </rejectfont>
 </selectfont>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>MingLiU_HKSCS-ExtB</string></patelt>
  </rejectfont>
 </selectfont>
 <selectfont>
  <rejectfont>
  <patelt name="family"><string>PMingLiU-ExtB</string></patelt>
  </rejectfont>
 </selectfont>
</fontconfig>
``` 

My Windows OS  is configured for the **English(United States) - US**  as language and **United States** for region.

Is this normal for Windows ? or it's a bug ?

**Comment by lovell** [comment]
> font-family="Times,serif" 

It looks like fontconfig has decided that the most appropriate font installed on this system to use for "Times" or "serif" is "MingLiU-ExtB", which does claim to support a lang of "en".

```
Best score 0 0 0 0 0 0 0 1e+99 0 0 1e+99 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2.14702e+12
Pattern has 27 elts (size 32)
	family: "MingLiU-ExtB"(w) "ç´°æ˜Žé«”-ExtB"(w)
	familylang: "en"(w) "zh-tw"(w)
	style: "Regular"(s)
	stylelang: "en"(w)
```
Is there a font installed called "Times"? If not, perhaps modify the `font-family` property to more closely match the expected output font.

**Comment by lovell** [comment]
@yoopyman Were you able to make any progress with this?

**Comment by yoopyman** [comment]
The SVG that I convert is generated by the library https://github.com/hpcc-systems/hpcc-js-wasm and I don't want to change manually the font in the SVG after is generated. It seems that by default the library generates the SVG with the font "Times" in it.
There are to possibilities to somehow solve the font problem: 
1) I changed in Graphviz script the font not to be the default one  which is strange that is set by default on "Times" if no font is set.
2) Set  the FontConfig in order to eliminate the fonts that are not ok.

Let assume that the "Times" font (which is a problem, but the reason is explained above) doesn't exist and the FontConfig tries 
to find a similar font it's very strange that it finds first all the zh-tw fonts (see above the list of all forbidden fonts) and only after that it searches the normal fonts: Arial,.... ("en").
I think there is some hard-coding in the FontConfig that prioritize somehow zh-tw fonts. I don't know if this happens also on Linux but on Windows it behaves like this. I have tested on multiple machines that are set on US with no relation to zh-tw and still prioritizes the zh-tw fonts first.

I fell that the problem is still there, but for now I will use the configuration of FontConfig to forbid the "strange"/zh-tw fonts  and it works for me.


Thank you for the support. The ticket can be closed.
